dev/src/glloadiqm.cpp

```diff
@@ -155,6 +155,7 @@ static const char **textures = nullptr;
 static iqmjoint *joints = nullptr;
 static iqmpose *poses = nullptr;
 static iqmanim *anims = nullptr;
+static const char **anim_names = nullptr;
```

Comment:
Not your problem, but one day we should stick all these globals in a class, hah!